guide.css

```diff
@@ -10,9 +10,35 @@ img{
 }
 
 .sectionTitle{
+	background-color: #FBEFD5;
+	border:none;
```

Comment:
border:  none is redundant and can be removed i think

Reply:
Removed in cfd429c18c1e60c75045ad0874b44a72bfe6769b